src/lifetimes/borrow-one.md

```diff
@@ -68,10 +68,16 @@ fn main() {
   enforcing that the function adheres to the contract set by the function's
   signature.
 
-- The "help" note in the error notes that we can add a lifetime bound `'b: 'a`
-  to say that `'b` will live at least as long as `'a`, which would then allow us
-  to return `query`. On the next slide we'll talk about lifetime variance, which
-  is the rule that allows us to return a longer lifetime when a shorter one is
-  expected.
+# More to Explore
+
+- The "help" message in the error notes that we can add a lifetime bound
+  `'b: 'a` to say that `'b` will live at least as long as `'a`, which would then
+  allow us to return `query`. This is an example of lifetime "variance", which
```

Comment:
This is lifetime _subtyping_; variance only has to do with the way the subtyping relation is reversed for function argument types relative to function return types.